userbot/modules/rastick.py

```diff
@@ -0,0 +1,58 @@
+import re
+import random
+from userbot import CMD_HELP, bot
+from userbot.events import register
+
+
+EMOJI_PATTERN = re.compile(
+    "["
+    "\U0001F1E0-\U0001F1FF"  # flags (iOS)
+    "\U0001F300-\U0001F5FF"  # symbols & pictographs
+    "\U0001F600-\U0001F64F"  # emoticons
+    "\U0001F680-\U0001F6FF"  # transport & map symbols
+    "\U0001F700-\U0001F77F"  # alchemical symbols
+    "\U0001F780-\U0001F7FF"  # Geometric Shapes Extended
+    "\U0001F800-\U0001F8FF"  # Supplemental Arrows-C
+    "\U0001F900-\U0001F9FF"  # Supplemental Symbols and Pictographs
+    "\U0001FA00-\U0001FA6F"  # Chess Symbols
+    "\U0001FA70-\U0001FAFF"  # Symbols and Pictographs Extended-A
+    "\U00002702-\U000027B0"  # Dingbats
+    "]+")
+
+
+def deEmojify(inputString: str) -> str:
+    return re.sub(EMOJI_PATTERN, '', inputString)
+
+
+@register(outgoing=True, pattern="^.rastick(?: |$)(.*)")
+async def rastick(animu):
+    text = animu.pattern_match.group(1)
+    if not text:
+        if animu.is_reply:
+            text = (await animu.get_reply_message()).message
+        else:
+            await animu.answer("`No text given, hence no stickers.`")
+            return
+    animus = [1, 2, 3, 4, 5, 6, 7, 8, 9, 10,
+             11, 12, 13, 14, 15, 16, 17, 18, 
+             19, 20, 21, 22, 23, 24, 25, 26, 
+             27, 28, 29, 30, 31, 32, 33, 34, 
+             35, 36, 37, 38, 39, 40, 41, 42, 
+             43, 44, 45, 46, 47, 48, 49, 50, 
+             51, 52, 53, 54, 55, 56, 57, 58, 
+             59, 60, 61, 62, 63]
+    sticcers = await bot.inline_query(
+        "stickerizerbot", f"#{random.choice(animus)}{(deEmojify(text))}")
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Standard pseudo-random generators are not suitable for security/cryptographic purposes.](https://app.codacy.com/manual/MoveAngel/One4uBot/pullRequest?prid=5574764)